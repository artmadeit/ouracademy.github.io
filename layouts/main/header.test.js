import Header from './header'
import { render } from 'enzyme'

describe('<Header>', () => {
  const component = render(<Header absolute={true} />)

  it('should go to home when click on the brand name', () => {
    expect(component.find('.navbar-brand').prop('href')).toBe('/')
  })

  it('should contain a nav menu ', () => {
    const links = component.find('.navbar-right a')
    expect(links).toMatchSnapshot()
  })
})
