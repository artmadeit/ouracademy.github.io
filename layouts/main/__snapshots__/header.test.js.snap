// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> should contain a nav menu  1`] = `
Array [
  <a
    href="/about"
  >
    Nosotros
  </a>,
  <a
    href="/tags"
  >
    Tags
  </a>,
]
`;
